app/i18n.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * i18n.js
+ *
+ * This will setup the i18n language files and locale data for your app.
+ *
+ */
+import { addLocaleData } from 'react-intl';
+
+import enLocaleData from 'react-intl/locale-data/en';
+import deLocaleData from 'react-intl/locale-data/de';
+
```

Comment:
maybe we should export a `defaultLocale` here and replace hardcoded 'en' strings in provider setup